src/cascadia/TerminalApp/SuggestionsControl.cpp

```diff
@@ -1048,6 +1173,7 @@ namespace winrt::TerminalApp::implementation
         _space = space;
 
         const til::size actualSize{ til::math::rounding, ActualWidth(), ActualHeight() };
+        const til::size descriptionSize{ til::math::rounding, _descriptionsBackdrop().ActualWidth(), _descriptionsBackdrop().ActualHeight() };
```

Comment:
This addition doesn't seem used anywhere.

Also, the `actualSize` above should probably similarly be just assignments to two variables so we don't lose the sub-DIP-precision, I think?